.changeset/popular-eggs-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/view-components": patch
```

Comment:
This is a breaking change in `auto-check-element`, so maybe it should be breaking for PVC too?

Reply:
Yep I think so